modules/sdk-core/src/bitgo/baseCoin/iBaseCoin.ts

```diff
@@ -364,7 +364,11 @@ export type SignedTransaction =
   | FullySignedTransaction
   | SignedTransactionRequest;
 
-export type SignedMessage = SignedTransactionRequest | string;
+export interface SignedMessage {
+  txHash: string;
```

Comment:
change field name to `singedMessage` - txHash is not relevant for message signing

Reply:
That may create issues as Front end is expecting this nomenclature.